evennia/utils/logger.py

```diff
@@ -19,8 +19,9 @@
 import time
 from datetime import datetime
 from traceback import format_exc
-from twisted.python import log
+from twisted.python import log, logfile
 from twisted.internet.threads import deferToThread
+from django.conf import settings
```

Comment:
There is a reason the settings file was not loaded in the header of the logger module - the reason is to keep the logger as separate as possible, so that it can start even in states when django is not yet initiated.